python/cxx/eclipse_io.cpp

```diff
@@ -13,29 +20,268 @@ namespace py = pybind11;
 
 namespace {
 
-py::array get_vector(Opm::EclIO::EclFile * file_ptr, std::size_t array_index) {
+using npArray = std::tuple<py::array, Opm::EclIO::eclArrType>;
+using EclEntry = std::tuple<std::string, Opm::EclIO::eclArrType, long int>;
```

Comment:
any reason why this is a long int? it's icky for portability.

Reply:
As part of supporting huge arrays , PR #1587 "Adding support for larger EclFile arrays ", EclEntry was changed in EclFile.hpp from int to long int

```C++
using EclEntry = std::tuple<std::string, eclArrType, long int>;
```

Reply:
well as discussed in email windows has a surprise for you. it's better to use int64_t.

Reply:
I'm planning to go through the EclIO classes and replace long int with int64_t and from unsigned long int with uint64_t.  

I will preferre to do this in a separate PR. I will start with this when this is completed.